ocaml/xapi-storage-script/main.ml

```diff
@@ -352,6 +352,8 @@ end
 
 let _nonpersistent = "NONPERSISTENT"
 
+let _vdi_mirror_in = "VDI_MIRROR_IN"
```

Comment:
Regarding the commit
> CP-45016: Implement checking of mirroring features 

What could be done to make the error less cryptic? maybe something along the error path to transform the error?